apps/vpnmanager/src/pages/index.tsx

```diff
@@ -10,8 +12,23 @@ export default function Home(props: Props) {
   return <Statistics data={data} />;
 }
 
-export async function getStaticProps() {
-  const data = await getStats({ query: { orderBy: "date DESC" } });
+export async function getServerSideProps(
+  context: GetSessionParams | undefined,
+) {
+  const yesterday = startOfYesterday();
+
+  const data = await getStats({
+    query: { orderBy: "date DESC", date: format(yesterday, "yyyy-MM-dd") },
+  });
+  const session = await getSession(context);
+  if (!session) {
```

Comment:
Same here, why are you fetching data _before_ you check if there is an active session?